### PR TITLE
tensorflow-base: add support for windows

### PR DESCRIPTION
**Pull request by nehaljwani**

### CODE REVIEW

File: tensorflow-base-feedstock/recipe/bld.cmake.bat [hunk 1]
```diff
@@ -0,0 +1,26 @@
+cd tensorflow\contrib\cmake
```
Comment (nehaljwani) [inline tensorflow-base-feedstock/recipe/bld.cmake.bat]: Keeping this one here just for the sake of posterity 😅

File: tensorflow-base-feedstock/recipe/meta.yaml [hunk 2]
```diff
@@ -20,10 +20,15 @@ source:
     - use_installed_mkl.patch                         # [linux]
     # add support for glibc 2.11
     - support_glibc_2_11.patch
+    - 0001-jpeg.BUILD-Using-cpu-instead-of-android_cpu-17508.patch
+    - 0002-Remove-TF_NEED_KAFKA-from-configure.py-as-it-is-not-.patch
+    - 0003-Update-is_windows-to-allow-mingw-shells.patch
+    - 0004-Escape-regex-pattern-properly-before-using-it.patch
```
Comment (jjhelmus) [inline tensorflow-base-feedstock/recipe/meta.yaml]: Should we add a # [win] selector to these patches?

Comment (jjhelmus) [reply tensorflow-base-feedstock/recipe/meta.yaml]: sounds good to me

**Comment by jjhelmus** [comment]
Thanks @nehaljwani, great work.

**Pull request merged**